src/org/opensolaris/opengrok/history/AccuRevHistoryParser.java

```diff
@@ -92,13 +92,6 @@ History parse(File file, Repository repos) throws HistoryException {
                 Executor executor = repository.getHistoryLogExecutor(file);
                 executor.exec(true, this);
 
-                /*
```

Comment:
I don't really like mixing various thing in one pull request ...
but then I do it as well being too lazy to open a new branch, so I guess we both need to sit down and read and start using the git branching as in https://github.com/OpenGrok/OpenGrok/wiki/Developer-intro
(thnx for the perfect writeup @vladak ! )

Reply:
but I did make a new branch, so I must not be doing something else correctly.

Reply:
I think this might fix itself once the other pullrequest is merged